### ISSUE TITLE
Sort is not usable in server page-side-pagination

### ISSUE THREAD

**Issue by billzt** [billzt/btcotton#5]
see http://jsfiddle.net/4r6g4cfu/3/light/

**Comment by billzt** [comment]
gene search OK c88fd12d0069cd6d9ed7322e9e5ac7828447e891

**Comment by billzt** [comment]
Other search is deleted. 9285a76858cfec1a1df5afe3d65c506365ef73a2

**Comment by billzt** [comment]
Consider search for data array.

``` php
    if (isset($sort)) {
        function build_sorter($key, $order) {
            return function ($a, $b) use ($key, $order) {
                if ($order=='asc') {
                    return strnatcmp($a[$key], $b[$key]);
                }
                else {
                    return -1*strnatcmp($a[$key], $b[$key]);
                }
            };
        }
        usort($data, build_sorter($sort, $order));
    }
```

**Comment by billzt** [comment]
As the parameters "limit, offset, sort ,order" is defined only after bootstrap table loaded, we must store all data first.

**Comment by billzt** [comment]
Fixed a0a194b3a65fec19c704ea2e79f4269250ea8a0a
